app/controllers/orga/seasons_controller.rb

```diff
@@ -45,23 +45,13 @@ def destroy
   end
 
   # # switch_phase: enables developers to easily switch between time dependent settings in views
-  # by opening and closing the corresponding links in the nav bar
+  # by showing the corresponding links in the nav bar
   def switch_phase
     return if Rails.env.production?
-      case params[:option]
-      when 'Proposals'
-        Season::PhaseSwitcher.fake_proposals_phase
-      when 'Application'
-        Season::PhaseSwitcher.fake_application_phase
-      when 'CodingSummer'
-        Season::PhaseSwitcher.fake_coding_phase
-      when 'RealTime'
-        Season::PhaseSwitcher.back_to_reality
-      end
-    redirect_to orga_seasons_path, notice: "We time travelled into the #{params[:option]} phase"
+    Season::PhaseSwitcher.destined(params[:phase])
```

Comment:
Hm, I need to try this out in real code tomorrow. I don't see how Rails could be able to distinguish between symbols and strings in the params hash. It's parsed out of the HTTP request and I cannot imagine how this should work then... But I'll try it out.